### DISCUSSION TITLE
Ok, I think we will all be OK

### DISCUSSION THREAD

**Discussion by RavenChill-Dev** [3kh0/ext-remover#764] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
### Ok, I can answer yall question for real.
I have been developing a fake jailbreak hack for school chromebooks, and it actually works, well will work if I finish it.
I've been working on this cool USB-Reload hack for school, and what it does gets confusing:

1. First, it automatically blows past any management firewall to get into developer mode by using Sh1mmer, then begins using some of the files on the usb, it will run a cool program I made to type up commands in a bash terminal and developer mode(also thanks Sh1mmer)
2. Then after typing up said commands to speed up the process, I will then boot the chromebook back into the recovery screen to back to Sh1mmer.
3. After Sh1mmer loads, I will then boot from my Chrome OS latest USB, that is edited to a point with a AI firewall, it will basically be impossible for the school to remove it. The Chrome OS will include an VPN, no management, and completely will be reversible by using a simple button on the new OS. (There will be software transferred to the normal OS to switch back and forth easy)

I hope to finish this by the end of the year, but first I have to go get some more USBS since most of them keep blowing themselves up due to some of the programs I've been running LOL.
This will change everything and chrome os developers will be freaking out when this is out, because after its done, all chromebook is safe from their admins controlling them, and they will be free XD.
I absolutely HATE chrome OS for multiple reasons, one of which of it being so crappy and easily editable and to hack into. This whole program it basically just trying to prove a point. Schools, you been giving kids who can afford plenty of free food (well at least my state) and you get us chromebooks, at least get us  on windows, not just to the special kids.
So yeah chrome admins, I dont give a shit about you having the most "secure" software, nobody gives a shit if a 13 year old kid can just blast past through the restrictions. *Mic drop*

**Comment by FancyBoiL1** [comment]
Proof? Proof?

Even Bigger question; Did anyone ask?

**Reply by RavenChill-Dev** [reply]
Yes.

**Reply by RavenChill-Dev** [reply]
100s people say chrome OS is shit.

**Reply by RavenChill-Dev** [reply]
Well, proof is here. Dont make your own OS's open source kids, be like Microsoft.

**Reply by MechaXYZ** [reply]
@RavenChill-Dev every linux distro:

**Comment by 3kh0** [comment]
Any proof? I am not saying you are posing, but there are too many skids to tell anymore

**Reply by RavenChill-Dev** [reply]
Hi @3kh0 I would just like to say thanks to everything you contributed to saying Chrome OS is trash, and it is. As for proof, I will be sending pictures of the bash commands, the OS, and my modded SH1MMER Ui soon, its just that I need some more time. I understand how Chrome OS works and how the browser and Bash commands. I hope you just stick with what I am saying. And i'm not some skid, i'm an 13 year old developer who has already made their own proxies, a games website (Still in Dev) and already made an AI Hacker who can basically break most server computers. I hope you just believe me.

**Reply by FancyBoiL1** [reply]
"Just believe me bro" is something people say on the internet when they actually don't have anything.

**Reply by RavenChill-Dev** [reply]
Who asked you? Have you made your own AI before? Do you even know how to code? Or are you saying things becuase your jealous.

**Reply by RavenChill-Dev** [reply]
You should just shut up, your being rude to me and proberly other people for no reason. @FancyBoiL1 stfu. You really need to.

**Reply by RavenChill-Dev** [reply]
All you make is "Something, But better" or "Im better".
Its rude.

**Reply by FancyBoiL1** [reply]
Have you actually even made your own AI? Do YOU even actually know how to code? I'm being rude to you for your baseless claims of grandeur. At least I am actually making something.

**Reply by RavenChill-Dev** [reply]
Bro, I know how to code multiple languages bro. I can do HTMl, CSS, Javascript for making webpages (I know a bit, not the best though), I know Java, C++. Node, and python. I have made my own AI, it chats to me through some of the files on my computer, and my claims are not baseless, I have a replit account you can search up. So yeah, your so called "Something" are prob some school project. Did you know I'm self taught? So now leave me tf alone if you got nothing else to say.

**Reply by FancyBoiL1** [reply]
I saw your replit with your amazing four projects, one is unfinished, one is taken from someone else, another is your first python script (made an hour ago which is broken¹), and an html project that is easily done with even elementary knowledge. I am also self taught (WOW so unusual), you should just go and prove that you are smart or cease responding to me if you want me to stop so badly. But I have seen people like you before, who think that they are smarter than everyone else but have nothing to show for it. You keep responding because you need to be right, without even showing any work to back it up. Where is your magical AI that talks to you? Show us the code then! Where is this magical Sh1mmer mod? Show us! You can't say your claims are not baseless when you back up only the most basic of knowledge. You want to know someone who can brag, and whose work shows credibility? 3kh0. That's who. He has credibility, I would never argue with him. Where is your credibility?

1: 
Options: left/right/backward/forward
forward
Traceback (most recent call last):
  File "/home/runner/494280430/main.py", line 22, in <module>
    meh()
NameError: name 'meh' is not defined
exit status 1

**Comment by sussybalsin2020** [comment]
@FancyBoiL1 stfu

**Reply by RavenChill-Dev** [reply]
Who you talking to?

**Reply by sussybalsin2020** [reply]
the guy who is arguing with you

**Reply by RavenChill-Dev** [reply]
Ok. TY, that guy is rude.

**Reply by RavenChill-Dev** [reply]
Lol, i'm sick (very) , I didn't even see the name.

**Reply by sussybalsin2020** [reply]
i hope u get better

**Reply by RavenChill-Dev** [reply]
Thanks :)
